pkg/daemon/ceph/osd/volume.go

```diff
@@ -211,6 +212,9 @@ func getCephVolumeOSDs(context *clusterd.Context, clusterName string) ([]oposd.O
 		var osdFSID string
 		isFilestore := false
 		for _, osd := range osdInfo {
+			if osd.Tags.ClusterFSID != cephfsid {
+				continue
```

Comment:
How about adding a log entry here to indicate that the OSD was skipped that belongs to another cluster? It would be good to indicate the OSD ID and mismatched fsid's in the message as well.

Reply:
Added logs while skipping OSD